### PR TITLE
Feature/29691

### PR DESCRIPTION
**Pull request by kjugi**
+ Add icons as files 
+ Replace sprite with icon 
+ Change link to sprite in preview
---
+ Add svg-sprite as deps 
+ Add task with icon sprite 
+ Change sprite name in previews
+ Rename icons like in `icons.config.js`
+ Rename sprite to `icons-sprite.svg` 
+ Delete unused icons
---
https://dev.snowdog.pro/issues/29691

### CODE REVIEW

File: components/general/_generalpreview.hbs [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <title>{{ title }}</title>
 </head>
 <body>
-    {{{ inline "components/general/icons/icon-sprite.svg" }}}
+    {{{ inline "public/images/icons-sprite.svg" }}}
     {{{ yield }}}
 </body>
 </html>
```
Comment (Igloczek) [inline components/general/_generalpreview.hbs]: You can remove this file.

File: public/images/icons-sprite.svg [hunk 2]
```diff
@@ -0,0 +1 @@
... repeatCount="indefinite"/></rect><rect x="46.5" y="40" width="7" height="20" rx="5" ry="5" fill="#f26322" transform="rotate(-108 39.102 65)"><animate attributeName="opacity" from="1" to="0" dur="0.7s" begin="0.48999999999999994s" repeatCount="indefinite"/></rect><rect x="46.5" y="40" width="7" height="20" rx="5" ry="5" fill="#f26322" transform="rotate(-72 29.354 65)"><animate attributeName="opacity" from="1" to="0" dur="0.7s" begin="0.5599999999999999s" repeatCount="indefinite"/></rect><rect x="46.5" y="40" width="7" height="20" rx="5" ry="5" fill="#f26322" transform="rotate(-36 3.835 65)"><animate attributeName="opacity" from="1" to="0" dur="0.7s" begin="0.63s" repeatCount="indefinite"/></rect></symbol><symbol viewBox="0 0 13 25" id="slider-left"><path fill-rule="evenodd" d="M15.06 3.04L4.1 14l10.96 10.96a1.5 1.5 0 0 1-2.12 2.12L.917 15.06a1.502 1.502 0 0 1 0-2.12L12.938.92a1.5 1.5 0 0 1 2.123 2.12z"/></symbol><symbol viewBox="0 0 13 25" id="slider-right"><path fill-rule="evenodd" d="M14.582 14.575L2.586 26.582a1.496 1.496 0 0 1-2.117 0 1.5 1.5 0 0 1 0-2.12l10.937-10.946L.47 2.568a1.5 1.5 0 0 1 0-2.12 1.496 1.496 0 0 1 2.116 0l11.996 12.008a1.5 1.5 0 0 1 0 2.12z"/></symbol></svg>
```
Comment (Igloczek) [inline public/images/icons-sprite.svg]: This file has to be gitignored.

File: package.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
     "gulp-plumber": "^1.1.0",
     "gulp-postcss": "^6.3.0",
     "gulp-sass": "^3.1.0",
-    "gulp-sourcemaps": "^2.4.1"
+    "gulp-sourcemaps": "^2.4.1",
+    "gulp-svg-sprite": "~1.3.6"
```
Comment (Igloczek) [inline package.json]: Let's change all `^` to `~`.

File: components/general/_generalpreview.hbs (hunk 1 above)
Comment (kjugi) [inline components/general/_generalpreview.hbs]: but if I remove this file we don't have one main preview and it looking bad when you start fractal and go into the general. The best solution will create main preview file (copy this from `block` dir)

File: public/images/icons-sprite.svg (hunk 2 above)
Comment (kjugi) [inline public/images/icons-sprite.svg]: Added file in `.gitignore`

File: package.json (hunk 3 above)
Comment (kjugi) [inline package.json]: Changed

File: components/general/_generalpreview.hbs (hunk 1 above)
Comment (Igloczek) [inline components/general/_generalpreview.hbs]: Remove this file and `components/blocks/_preview.hbs`, but add exactly same code as `components/_preview.hbs`.

File: public/images/icons-sprite.svg [hunk 4]
```diff
@@ -0,0 +1 @@
... dur="0.7s" begin="0.27999999999999997s" repeatCount="indefinite"/></rect><rect x="46.5" y="40" width="7" height="20" rx="5" ry="5" fill="#f26322" transform="rotate(180 50 65)"><animate attributeName="opacity" from="1" to="0" dur="0.7s" begin="0.35s" repeatCount="indefinite"/></rect><rect x="46.5" y="40" width="7" height="20" rx="5" ry="5" fill="#f26322" transform="rotate(-144 45.126 65)"><animate attributeName="opacity" from="1" to="0" dur="0.7s" begin="0.41999999999999993s" repeatCount="indefinite"/></rect><rect x="46.5" y="40" width="7" height="20" rx="5" ry="5" fill="#f26322" transform="rotate(-108 39.102 65)"><animate attributeName="opacity" from="1" to="0" dur="0.7s" begin="0.48999999999999994s" repeatCount="indefinite"/></rect><rect x="46.5" y="40" width="7" height="20" rx="5" ry="5" fill="#f26322" transform="rotate(-72 29.354 65)"><animate attributeName="opacity" from="1" to="0" dur="0.7s" begin="0.5599999999999999s" repeatCount="indefinite"/></rect><rect x="46.5" y="40" width="7" height="20" rx="5" ry="5" fill="#f26322" transform="rotate(-36 3.835 65)"><animate attributeName="opacity" from="1" to="0" dur="0.7s" begin="0.63s" repeatCount="indefinite"/></rect></symbol></svg>
```
Comment (Igloczek) [inline public/images/icons-sprite.svg]: Remove this file and add to `.gitignore`

**Comment by Igloczek** [comment]
Remember to resolve conflicts.

File: components/general/_generalpreview.hbs (hunk 1 above)
Comment (kjugi) [inline components/general/_generalpreview.hbs]: Ok, removed and created main `_preview.hbs` file

File: public/images/icons-sprite.svg (hunk 4 above)
Comment (kjugi) [inline public/images/icons-sprite.svg]: Removed and `.gitignore` updated

**Pull request merged**